cli/src/test/java/com/box/l10n/mojito/cli/command/QuartzJobsDeleteCommandTest.java

```diff
@@ -59,11 +44,16 @@ public void testDeleteAllDynamicJobs() throws Exception {
     getL10nJCommander().run("quartz-jobs-view");
     assertTrue("Should show 1 job", outputCapture.toString().contains("AJob_" + testJobName1));
     assertTrue("Should show 1 job", outputCapture.toString().contains("AJob_" + testJobName2));
-    assertEquals(2L, quartzService.getDynamicJobs().size());
 
+    Predicate<String> jobsFilter =
+        job -> job.contains("AJob_" + testJobName1) || job.contains("AJob_" + testJobName2);
+
+    assertEquals(2L, quartzService.getDynamicJobs().stream().filter(jobsFilter).count());
     getL10nJCommander().run("quartz-jobs-delete");
-    getL10nJCommander().run("quartz-jobs-view");
```

Comment:
Should the view have been removed?

Reply:
Yes, I don't think it should have been there in the first place as it just prints out the jobs to the CLI and the assertion afterwards had nothing to do with using this output.